src/com/amazon/ionelement/api/IonElement.kt

```diff
@@ -99,7 +99,7 @@ import java.math.BigInteger
 interface IonElement {
 
     /** The Ion data type of the current node.  */
-    val type: IonType
+    val type: ElementType
```

Comment:
This is technically backwards incompatible, should we model this as a major version change in the package?

Reply:
I don't know that anyone has taken a dependency on this yet, but it's possible.  So, yes, this will require a major version bump.

Reply:
But we normally bump versions before a release.  Are you suggesting we do that now?